io/subsystem/src/main/java/org/wildfly/extension/io/WorkerResourceDefinition.java

```diff
@@ -146,33 +152,31 @@ public void registerCapabilities(ManagementResourceRegistration resourceRegistra
     public void registerAttributes(ManagementResourceRegistration resourceRegistration) {
         resourceRegistration.registerReadWriteAttribute(WORKER_TASK_MAX_THREADS,
                 new WorkerReadAttributeHandler(WORKER_TASK_MAX_THREADS.getOption()),
-                new WorkerWriteAttributeHandler() {
+                new WorkerWriteAttributeHandler(ATTRIBUTES){
```

Comment:
This can be WORKER_TASK_MAX_THREADS which is a minor optimization, saving storing some irrelevant attributes in the handler's attributeDefinitions map.